Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,124 @@
+- The teams I work with are consistently winning. They chose ambitious targets, managed scope and expectations effectively, and delivered customer value frequently with customer adoption trends going in the right direction.
+- I identified a problem that was timely, important, and impactful, and worked closely with product, engineering, and executive stakeholders to articulate and refine a solution and delivery plan.
+- I identified a significant problem that was increasing cognitive load for engineers resulting in multiple defects, then created a lasting best-fit solution that aligned with our strategy and solved the problem.
+- I identified a pattern of organisational dysfunction that was causing poor outcomes, shared my observations with my leadership team, and escalated the signal effectively, resulting in the dysfunction being corrected and collectively getting better outcomes.
+- I regularly shared timely information in the right forums, like Slack and R&D Weekly, to amplify the work being done by my team(s), to detect misalignment or surprises, and to build confidence in our collective delivery plans.
 - Instead of simply patching or extending an existing solution that was not fit for purpose anymore, my contribution opened up a whole new area of strategic possibility.
 - I regularly participated in our code review and interview processes and provided feedback on how they could be improved.
 
 </details>
 
 ## 🏆 Customer Success
 
-- I have a deep understanding of our products and product strategy.
+- I have a broad understanding of Octopus product offerings and product strategy.
+- I have a deep understanding my group's product offerings and product strategy.
 - I understand what technical challenges are most impacting our customers at present, and how my own plans and the Octopus technical strategy seek to solve them.
 - I am able to anticipate and adapt systems and practices to changes in load, usage, and customer requirements.
-- I work closely with our product team to ensure continued product/market fit.
+- I work closely with my product managers and designers to ensure the best outcomes for our customers and continued product/market fit.
 - I find the balance between doing it right and getting it done.
+- I am an expert at managing scope, finding ways to help my teams consistently deliver the high value to customers at speed.
+- I hold myself and my teams to a high standard of quality, with a specific focus on reliability, usability, and functionality.
 - I am responsive to changes in product priorities, keeping multiple teams focused on important, high-value efforts.
 - I inform product strategy by helping senior leadership understand the organisation's engineering capabilities and technical strategy.
+- I help people from other departments, primarily support, customer success, revenue, and marketing, when my expertise will deliver better outcomes for our customers.
```

Comment:
```suggestion
- I collaborate with people from other departments, primarily support, customer success, revenue, and marketing, when my expertise will deliver better outcomes for our customers.
```

Suggest framing the interaction as neutral/symmetric, not dependent (i.e. "they" can help "us", too! :relaxed:).